spec/core/ics-004-channel-and-packet-semantics/UPGRADES.md

```diff
@@ -87,18 +87,27 @@ function restorePath(portIdentifier: Identifier, channelIdentifier: Identifier):
 }
 ```
 
-#### UpgradeError Path
+#### CurrentUpgradeSequencePath
```

Comment:
I'd remove "current", i.e., `UpgradeSequencePath`. Also make it consistent throughout the file.